pkg/recipes/terraform/config/config.go

```diff
@@ -96,24 +97,32 @@ func (cfg *TerraformConfig) Save(ctx context.Context, workingDir string) error {
 	// JSON configuration syntax for Terraform requires the file to be named with .tf.json suffix.
 	// https://developer.hashicorp.com/terraform/language/syntax/json
 
-	// Convert the Terraform config to JSON
-	jsonData, err := json.MarshalIndent(cfg, "", "  ")
-	if err != nil {
+	// Create a buffer to write the JSON to
+	buf := &bytes.Buffer{}
+	enc := json.NewEncoder(buf)
+	enc.SetEscapeHTML(false)
+	enc.SetIndent("", "  ")
```

Comment:
Why are there two spaces here? Should we add a comment here so that the next person to work on this part could know better?

Reply:
added comment